designsafe/apps/api/licenses/views.py

```diff
@@ -0,0 +1,25 @@
+from designsafe.apps.api.views import BaseApiView
+from designsafe.apps.api.mixins import SecureMixin
+from designsafe.libs.common.decorators import profile as profile_fn
+from django.contrib.auth import get_user_model
+from django.http.response import HttpResponseForbidden, HttpResponseNotFound
+from django.http import JsonResponse
+from django.apps import apps
+from designsafe.apps.data.models.agave.util import AgaveJSONEncoder
+
+
+class LicenseView(SecureMixin, BaseApiView):
+    @profile_fn
+    def get(self, request, app_name, username):
+        if not request.user.is_staff:
+            return HttpResponseForbidden()
+        
+        if app_name not in ['MATLAB','LSDYNA']:
+            return HttpResponseNotFound()
+        app_license = apps.get_model('designsafe_licenses', '{}License'.format(app_name))
```

Comment:
Maybe we use `try/except` here, and we can raise `HttpResponseNotFound()` on the `except` if the `get_model` raises an exception. That way we keep this decoupled from the models we've added in the licenses app, in case it gets expanded